### ISSUE TITLE
.NET 6 Preview 4 SDK depends on Preview 5 bits

### ISSUE THREAD

**Issue by danroth27** [dotnet/aspnetcore#31969]
Repro steps:
- Install .NET SDK 6.0.100-preview.4.21219.6
- `dotnet new blazorwasm -ho`

Expected result: Generated template depends on Preview 4 packages
Actual result: Generated template depends on Preview 5 packages

```xml
<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">

  <PropertyGroup>
    <TargetFramework>net6.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="6.0.0-preview.5.21217.2" />
    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="6.0.0-preview.5.21217.2" PrivateAssets="all" />
  </ItemGroup>

  <ItemGroup>
    <ProjectReference Include="..\Shared\ConwayBlazor.Shared.csproj" />
  </ItemGroup>

</Project>
```

**Comment by davidfowl** [comment]
Ah the crossing of streams 😢

**Comment by javiercn** [comment]
Thanks Dan

I think this might be "temporary"?

**Comment by mkArtakMSFT** [comment]
@dotnet/aspnet-build folks, can you confirm whether this is temporary or whether something will need to be fixed to address this? And if so, that'll have to be done for preview 4.

**Comment by Pilchie** [comment]
From an internal thread, this was because the SDK repo branched slightly late. @mmitche expects it to get cleaned up in the next build or two.

**Comment by dougbu** [comment]
Unassigning myself because there's no action we can take but wait

**Comment by pranavkm** [comment]
Resolved as part of 6.0.100-preview.4.21222.3.
